### PR TITLE
Yathinsp patch 1

### PR DESCRIPTION
**Pull request by yathinsp**

### CODE REVIEW

File: public/javascript/script.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Promise.all([
+    faceapi.nets.faceRecognitionNet.loadFromUri('/models'),
+    faceapi.nets.faceLandmark68Net.loadFromUri('/models'),
+    faceapi.nets.ssdMobilenetv1.loadFromUri('/models') //heavier/accurate version of tiny face detector
+]).then(start)
+
+function start() {
+    document.body.append('Models Loaded')
+
+    navigator.getUserMedia(
```
Comment (dhruvshettty) [inline public/javascript/script.js]: This method is deprecated. Use the newer method. Read more about it [here](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getUserMedia).
```suggestion
    navigator.mediaDevices.getUserMedia(
```

File: public/javascript/script.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                return faceMatcher.findBestMatch(d.descriptor)
+            })
+            results.forEach( (result, i) => {
+                const box = resizedDetections[i].detection.box
+                const drawBox = new faceapi.draw.DrawBox(box, { label: result.toString() })
+                console.log(result)
+                drawBox.draw(canvas)
+                setTimeout(() => {
+                    if(result.label == "unknown") {
+                        document.location = "http://localhost:3000/verification/alert-owner"
```
Comment (dhruvshettty) [inline public/javascript/script.js]: The endpoint should be changed.
```suggestion
                        document.location = "http://localhost:3000/verification/owner-verify"
```

File: public/javascript/script.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    } else {
+                        document.location = "http://localhost:3000/verification/success"
+                    }
+                },3000)
```
Comment (dhruvshettty) [inline public/javascript/script.js:61]: This timeout is unnecessary? You could probably increase the timeout period for the face recognition and run the routing callback whenever a result is generated.
This is just a performance improvement. We could skip this if everything else works.

File: public/models/d.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (dhruvshettty) [inline public/models/d.txt]: What's this for?

**Review by dhruvshettty — CHANGES_REQUESTED** [review]
Please resolve the pending comments I've made. You can delete your personal images (doesn't make sense making it public) and instead put a placeholder file so that the `public/labeled_images` directory shows up in git.

File: public/javascript/script.js (hunk 3 above)
Comment (yathinsp) [reply public/javascript/script.js:61]: if we dont add timeout it ll display the result i.e., re-route immediately after 100ms, the longer the models runs accurate is the detection

File: public/models/d.txt (hunk 4 above)
Comment (yathinsp) [reply public/models/d.txt]: i deleted it now it was created as a dummy to create a folder

**Review by dhruvshettty — APPROVED** [review]

**Review by dhruvshettty — APPROVED** [review]
Closes Issue #1

**Pull request merged**